assembl/semantic/jsonld_reader.py

```diff
@@ -1,151 +0,0 @@
-"""Import JSON-LD into assembl."""
```

Comment:
Does the removal of virtuoso imply inability to have our jsonld import and export/representation features work properly? Or is this code removed because it was not used anymore?

Reply:
We had lost JSON-LD with the migration to postgres long ago. Reestablishing JSON-LD would be a major project. So this is part of trimming unused code.

Reply:
ok